### DISCUSSION TITLE
facing a issue in the installation

### DISCUSSION THREAD

**Discussion by kumaran1223** [pieces-app/support#544] [Q&A]
![Screenshot (120)](https://github.com/user-attachments/assets/93938b60-8005-4432-b4d8-1b26595a18a5)

**Comment by mack-at-pieces** [comment]
Hey @kumaran1223 Thanks so much for raising this. I've created a GitHub issue regarding this located here: https://github.com/pieces-app/support/issues/545

I'll continue the troubleshooting over there 😄
